config/q_it.rb

```diff
@@ -0,0 +1,8 @@
+# This class contains methods and constants to be used throughout 
+# the application among all classes and modules.
+class QIt
+ 
+  def self.root
```

Comment:
Good.
Just make changes everywhere - use `QIt.root` as the base to all other locations.

Reply:
I think I have made those changes already. Do you see anywhere else we could use it?